process/block/preprocess/smartContractResults.go

```diff
@@ -516,9 +516,12 @@ func (scr *smartContractResults) ProcessMiniBlock(
 
 	defer func() {
 		if err != nil {
-			scr.gasHandler.RemoveGasProvided(processedTxHashes)
-			scr.gasHandler.RemoveGasRefunded(processedTxHashes)
-			scr.gasHandler.RemoveGasPenalized(processedTxHashes)
+			for _, hash := range processedTxHashes {
+				//TODO: Remove it or set log level to Trace
```

Comment:
resolve TODO?

Reply:
We are hunting a rare situation, so I will keep this log on Trace, which anyway should not appear too often, just to catch this error which seems to come from sc execution